backends/tfhe-cuda-backend/cuda/src/integer/multiplication.cuh

```diff
@@ -20,28 +20,11 @@
 #include <fstream>
 #include <iostream>
 #include <omp.h>
+#include <queue>
```

Comment:
Do we really need this new include?

Reply:
yes we use it to calculate final degrees